tests/server/lib/dnshook.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+// Replace the builtin `dns.lookup` with one that will divert lookups of
+// `alias` to a test `elb`. Useful for testing deployments before going live,
+// specifically for `tests/server/l10n-entrained`.
+//
+// Caveats:
+// - the `got` module appears to wind up using `dns.lookup`, but this is a
```

Comment:
I'm not quite following you here. By "fragile assumption", I meant that that is how it works now, but there's no contract that guarantees that.